### ISSUE TITLE
setPerPage bug

### ISSUE THREAD

**Issue by harauzku02** [nhn/tui.grid#848]
**Describe the bug**
Bug on setPerPage call

![K-20191219-672075](https://user-images.githubusercontent.com/59047461/71162764-1ff46700-228f-11ea-95fa-5eb6f387c803.png)
![K-20191219-672245](https://user-images.githubusercontent.com/59047461/71162765-21be2a80-228f-11ea-97e0-6e256269741a.png)
![K-20191219-672368](https://user-images.githubusercontent.com/59047461/71162769-22ef5780-228f-11ea-8bfd-9593819f097d.png)
![K-20191219-672442](https://user-images.githubusercontent.com/59047461/71162770-24208480-228f-11ea-9537-aaaff633b3e7.png)

**To Reproduce**
Initial Source Code Loading Time 
(최초 로딩 시 )
```js
this.gridOptions = {
  pageOptions: {
    perPage: 10,
    useClient: true,
  },
};
```
10 pages per page, 13 pages total.
In this state, you can call on ChangeCount() to set the number of exposures per page, and then set the setPerPage (pageCnt) in the page.
The list is not changed by 1 page, but you must click on the page after 1 page (2 pages or any page)
The number of lists will be changed.

(페이지당 10개씩 총 13페이지가 생성이 되어있는데
해당 상태에서 페이지당 노출 개수를 세팅하는 onChangeCount() 를 호출하여 그 안에서 changePerPage(pageCnt) 를 호출 후 setPerPage 를 세팅하는데 pagination 은 변경이 되는데 
리스트는 1페이지는 변경이 되지않고 1페이지 이후 페이지(2페이지나 아무페이지)를 클릭해야 
리스트 개수가 변경이 됩니다.)

**Expected behavior**

**Screenshots**
With the number of pages selected next to the "방문고객데이터" is 30 pages, the painting is configured normally, but the page list does not change.

("방문고객데이터" 옆에 선택된 페이지당 개수가 30개 인 상태에서 pagination 은 정상적으로 구성되나 페이지 리스트는 변하지 않음)
![image](https://user-images.githubusercontent.com/59047461/71162189-1c141500-228e-11ea-801e-84d58f522e69.png)

Only after clicking on another page will there be 30 lists as many as selected
(다른 페이지를 클릭 후에야 선택한 개수만큼 30개의 리스트가 나옴)
![K-20191219-663055](https://user-images.githubusercontent.com/59047461/71162301-4bc31d00-228e-11ea-9a1e-d6d3cf114bcf.png)

**Desktop (please complete the following information):**
 - OS: windows 10 home
 - Browser :  chrome
 - Version : "@toast-ui/vue-grid": "2.1.0",
    "tui-grid": "4.8.0",
    "tui-pagination": "3.3.3",

**Additional context**
If it's not a bug, can I refresh or rendering the list when I setPerPage?
(버그가 아니라면 grid 메소드 중에 setPerPage 할 때 리스트를 refresh 하거나 rendering 을 setPerPage 설정대로 할 수 있나요?)

**Comment by js87zz** [comment]
@harauzku02 
Please respect our policy that writing the form **in English** to share our issue to other people.
Change the title and content in English. 
It's helpful to give us detail example source in [codepen](https://codepen.io/) or [jsfiddle](https://jsfiddle.net/) for solving the issue. It's hard to understand your issue now.

**Comment by js87zz** [comment]
@harauzku02 
I've been checked this issue and It's internal bug about updating the height of grid on changing of page list. It will be fixed and after then i will comment here.
Thank you :)

**Comment by harauzku02** [comment]
@js87zz 
If you said bug, is there any other method or method to reload the grid or change the value of perPage when you call setPerPage?

**Comment by js87zz** [comment]
@harauzku02 
The data can be refreshed through calling `resetData` API, but it's not recommended.
We are planned to release bug-fix on next week or January.

**Comment by harauzku02** [comment]
@js87zz 
this.gridOptions = {
      pageOptions: {
        perPage: 10,
        useClient: true,
      },
      pagination: {
        options: {
          template: {
            page: '<button type="button" class="first" disabled>{{page}}</button>',
            moveButton: '<button type="button" class="prev" disabled>{{page}}</button>',
          },
        },
      },
    };
Like the code above, I tried to change the template to the same level as the grid page option, but it doesn't work. Is there any way I can change it?

**Comment by js87zz** [comment]
@harauzku02 
The mustache template is syntax for Vue, we don't support it and pagination option doesn't exist in grid option(refer [here](https://nhn.github.io/tui.grid/latest/Grid)).
Currently, There is no API to control page count.

**Comment by js87zz** [comment]
@harauzku02 
The bug has been fixed in `v4.8.1`(#852)
Thank you!
